.github/workflows/release.yml

```diff
@@ -15,3 +15,57 @@ jobs:
       with:
         workdir: /home/runner/work/inspektor-gadget/inspektor-gadget
         krew_template_file: .krew.yaml
+    - name: Update gadgets' version on artifact hub
+      id: bump_gadgets
+      if: github.repository == 'alban/inspektor-gadget'
+      run: |
+        echo "github_repository=${inspektor_gadget_version}" >> $GITHUB_OUTPUT
+        # Get the version without the "v" prefix
+        inspektor_gadget_version=$(curl --silent https://api.github.com/repos/${{ github.repository }}/releases/latest | jq .tag_name | tr -d '"v')
+
+        echo "version=${inspektor_gadget_version}" >> $GITHUB_OUTPUT
+
+        for PKGFILE in $(find gadgets/ -name artifacthub-pkg.yml) ; do
+            sed -i \
+                -e "s/^version:.*$/version: ${inspektor_gadget_version}/" \
+                -e "s/:latest/:v${inspektor_gadget_version}/g" \
```

Comment:
The first replace expression is fine.

But the second replace expression has a problem: it will work the first time because it will replace `:latest` by the correct release. But then at the second release, the tag will no longer be called `:latest` so the match will fail and it will not be updated.

I am not so sure anymore if we want to advertise the last release on Artifact Hub, or do we want to advertise the `latest` tag? Maybe `:latest` in the doc is better, and users can switch to a stable version if they want to. Then, I could just drop the second replace expression. What do you think?